tools/setup.py

```diff
@@ -124,6 +124,11 @@ def generate_gn_args(mode):
             tc = "//build_extra/toolchain/win:win_clang_x64"
             out += ['custom_toolchain="%s"' % tc, 'host_toolchain="%s"' % tc]
 
+    # Look for sccache; if found, set rustc_wrapper.
+    rustc_wrapper = find_executable("sccache")
+    if rustc_wrapper:
+        out += ['rustc_wrapper=%s' % gn_string(rustc_wrapper)]
```

Comment:
It seems this could result in a mixture of sccache and ccache if people have both installed? Maybe it would be better to detect and use sccache for cc_wrapper too?

Reply:
I think that's an interesting idea, however I didn't want to do it as this PR, as I suspect sccache may be a little slower than ccache.

Also it effectively makes travis build "from scratch" and i'd have had to wake up hourly last night and push the "restart build" button.

Tryout build: https://travis-ci.com/denoland/deno/builds/85958318
(feel free to het restart until it passes)